client/src/partials/budget/new/new_budget.js

```diff
@@ -0,0 +1,357 @@
+angular.module('bhima.controllers')
+.controller('NewBudgetController', [
```

Comment:
You should use the `$inject` property for readability and testability.  @sfount suggested it in #719.

Here's how to touch up the code to abide by this convention:
```js
angular.module('bhima.controllers')
.controller('NewBudgetController', NewBudgetController); // we will define this further down as a named function

NewBudgetController.$inject = [
  '$q', '$scope', '$http',  /* all other dependencies go here ... */
];

/**
*  New Budget Controller
*  This controller is responsible for creating a new budget.  Keep in mind to write useful comments in this space!
*  You do not need to discuss the parameters (since they are injected and can be lookup up as services), but make
*  sure you describe what the controller does.
*/
function NewBudgetController($q, $scope, $http, /* other dependencies ... */) {
  /* Awesome code */
}
```